README.md

```diff
@@ -72,8 +72,10 @@ Following this, various dependencies are installed depending on the following en
   version is installed (more info about LTS can be found
   [here](https://github.com/astropy/astropy-APEs/blob/master/APE2.rst#version-numbering).
 
-* ``$CONDA_DEPENDENCIES``: this should be a space-separated string of package
-  names that will be installed with conda.
+* ``$CONDA_DEPENDENCIES``: this should be a space-separated string of
+  package names that will be installed with conda. Version numbers of these
+  dependencies can be overriden/specified with the ``$${PACKAGENAME}_VERSION``
```

Comment:
I think just saying '``$PACKAGENAME_VERSION``, where ``PACKAGENAME`` is the name of the package to specify the version for (e.g. ``MATPLOTLIB_VERSION``)' would be fine (i.e. no need for double ``$``)